app/controllers/catalog_controller.rb

```diff
@@ -52,6 +52,14 @@ def search_service_context
     }
   end
 
+  # Should be equivalent to:
+  # ->(controller, field) { controller.can?(:access_staff_functions) }
+  # but was not able to get this syntax to work.
+  def can_see_all_search_facets?
+    can? :access_staff_functions
+  end
```

Comment:
ok no big deal to leave this, but let's change name to `access_staff_only_facets?`, or someth8ng liek that, what do you think? Sorry for being picky over naming.